### ISSUE TITLE
Error creating Taproot address from Ride The Lightning for on-chain funding of the LN node.

### ISSUE THREAD

**Issue by soyishikawa** [Ride-The-Lightning/RTL#1260]
**Describe the bug**
Error creating Taproot address from Ride The Lightning for on-chain funding of the LN node.

**To Reproduce**
1. Go to 'On-chain'
2. Click on 'Generate Address'
3. Select to 'Taproot (P2TR)'
4. See error: Only adddress segwit is admit, please chooso other option

**Screenshots**
![error taproot](https://github.com/Ride-The-Lightning/RTL/assets/46906582/a839a700-ffba-4ca5-9439-3262f1ea1011)

**Mainnet**
* Version of `RTL 0.14.0-beta`
* Version of core lightning/23.02.2
* Version of bitcoind/24.0.1
* Browser Tor 12.0.7 (based on Mozilla Firefox 102.12.0esr) (64-bit)
* Operating system (Windows/Ubuntu 22.04)

**Additional context**
Please check the error, beacuse is important to star use Taproot Address, more adoptiomn is neded.

Thask you.

**Comment by saubyk** [comment]
@ShahanaFarooqui this needs to be looked at. AFAIK, core lightning doesn't support p2tr output, so this shouldn't be an option for generating an address for core lightning

**Comment by BTCBellyButton** [comment]
Since CLN 23.08 Taproot addresses are supported.
The issue is now a bit different from the original:

![image](https://github.com/Ride-The-Lightning/RTL/assets/68573637/59e0e065-c34b-4d1d-a69d-e2f8b930d7b1)

I just tested it from the command line and it works:
```
lightning-cli newaddr p2tr
{
   "p2tr": "bc1pxj................................"
}

```
CLN 23.08.1
RTL 0.14.0-beta
API: 0.10.6

**Comment by ShahanaFarooqui** [comment]
Closing it with c-lightning-REST's PR #206.

**Comment by Frankenmint** [comment]
I'm still experiencing this issue. idk if a regression happened somewhere to cause this:

![image](https://github.com/Ride-The-Lightning/RTL/assets/7620177/fdaeb56d-5ca4-4aa5-9671-845fb5ef5cdc)

**Comment by ShahanaFarooqui** [comment]
@Frankenmint I am unable to reproduce it locally. Can you confirm:
* Your environment
* Version of `RTL`
* Version of `lnd`/`core lightning`/`eclair`
* Version of `btcd`, `bitcoind`, or other backend
* Browser & browser version
* Operating system (`uname -a` on *Nix)
* any other relevant environment details

**Comment by Frankenmint** [comment]
docker container ps output (scrubbed):

IMAGE                                           COMMAND                  NAMES
kukks/nnostr-relay:v0.0.18                      "dotnet Relay.dll"       nnostr-relay
btcpayserver/docker-gen:0.10.7                  "/usr/local/bin/dock…"   nginx-gen
btcpayserver/btcpayserver:1.13.0                "/app/docker-entrypo…"   generated_btcpayserver_1
saubyk/c-lightning-rest:0.10.6                  "/sbin/tini -g -- ./…"   generated-clightning_bitcoin_rest-1
btcpayserver/lightning:v23.08                   "/usr/bin/tini -g --…"   btcpayserver_clightning_bitcoin
btcpayserver/bitcoin:26.0                       "/entrypoint.sh bitc…"   btcpayserver_bitcoind
nicolasdorier/nbxplorer:2.5.2                   "dotnet NBXplorer.dll"   generated_nbxplorer_1
btcpayserver/cloudflared:2023.10.0              "cloudflared --no-au…"   cloudflared-tunnel
btcpayserver/docker-gen:0.10.7                  "/usr/local/bin/dock…"   tor-gen
btcpayserver/woocommerce:3.1.0                  "docker-entrypoint.s…"   generated-woocommerce-1
shahanafarooqui/rtl:0.14.1                      "/sbin/tini -g -- no…"   generated-bitcoin_rtl-1
mariadb:10.4                                    "docker-entrypoint.s…"   generated_mariadb_1
btcpayserver/tor:0.4.8.10                       "./entrypoint.sh tor"    tor
nginx:1.25.3-bookworm                           "/docker-entrypoint.…"   nginx
btcpayserver/postgres:13.13                     "/migrate-docker-ent…"   generated_postgres_1
btcpayserver/btcpayserver-configurator:0.0.21   "./docker-entrypoint…"   generated-configurator-1





browser?: you know chrome with the extra spicy ad blockers. here's another screen with my inspect console open if that helps (sorry if it hurts):

![image](https://github.com/Ride-The-Lightning/RTL/assets/7620177/9e3c6790-fcbf-4c4c-b8a1-e8234d4dfeda)



Linux SCRUBBED 6.5.0-28-generic #29-Ubuntu SMP PREEMPT_DYNAMIC Thu Mar 28 23:46:48 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux

**Comment by ShahanaFarooqui** [comment]
You are using c-lightning-REST v0.10.6 and the issue was resolved with release v0.10.7.
